katello-selinux-relabel

```diff
@@ -15,8 +15,9 @@
   /etc/cron.d/foreman* \
   /usr/lib/ruby/gems/1.8/gems/passenger-* \
   /usr/lib64/gems/ruby/passenger-*/agents \
-  /usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents
-
+  /usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents \
+  /var/lib/mongodb
+  
 # relabel SCL mod_passenger and foreman plugins if SCL is found
 [ -d /opt/rh/ruby193/ ] && /sbin/restorecon -ri $* \
```

Comment:
We moved our SCL to ```tfm``` so this is unused on modern installs.

Reply:
Right, let's fix it then. The correct path seems to be `/opt/theforeman/tfm/root/usr/share/gems/gems/passenger-4.0.18/bin/passenger` on my system.

Reply:
Actually, remove all passenger (and also non-katello) related things from here. This file should only have paths defined in katello `.fc` files!

Reply:
That's a good point. `foreman-selinux` would be a better place for it. Then `katello-selinux` could call the restore script from `foreman-selinux`.

Reply:
In foreman-selinux it's already fixed, no need of any PR there. I think the goal is to clean katello-selinux that's really all.